tests/exploratory_analyses/k_means_cluster_test.py

```diff
@@ -0,0 +1,30 @@
+import geopandas as gdp
+import pandas as pd
+import pytest
+
+from eis_toolkit.exceptions import EmptyDataFrameException
+from eis_toolkit.exploratory_analyses.k_means_cluster import k_means_clustering
+
+df = pd.DataFrame(
+    {
+        "Location": ["A", "B", "C", "D", "E", "F"],
+        "Latitude": [1, 2, 3, 4, 5, 6],
+        "Longitude": [-1, -2, -3, -4, -5, -6],
+    }
+)
+gdf = gdp.GeoDataFrame(df, geometry=gdp.points_from_xy(df.Longitude, df.Latitude), crs="EPSG:4326")
+
+
+def test_k_means_clustering_output():
+    """Test that k-means function assings data points into correct clusters."""
+    kmeans_gdf = k_means_clustering(data=gdf, number_of_clusters=2, random_state=0)
+    kmeans_labels = kmeans_gdf["cluster"]
+    expected_labels = [1, 1, 1, 0, 0, 0]
+    assert all([k == e for k, e in zip(kmeans_labels, expected_labels)])
+
```

Comment:
This is fine, but maybe slightly simpler would be to just `assert list(kmeans_labels) == expected_labels`. Just a thought